# YoutubeCommentsDataSet.csv exploration for fine tuning

In [1]:
import pandas as pd 
import numpy as np 

In [13]:
raw_data = pd.read_csv("https://raw.githubusercontent.com/ArrafMelon/Youtube-Video-Sentiment/main/data/raw_data/YoutubeCommentsDataSet.csv")
raw_data.head()

,Comment,Sentiment
0,lets not forget that apple pay in 2014 require...,neutral
1,here in nz 50 of retailers don’t even have con...,negative
2,i will forever acknowledge this channel with t...,positive
3,whenever i go to a place that doesn’t take app...,negative
4,apple pay is so convenient secure and easy to ...,positive


In [14]:
raw_data.shape

(18408, 2)

In [18]:
raw_data["Sentiment"].value_counts()

Sentiment
positive    11055
neutral      4504
negative     2318
Name: count, dtype: int64

## Data Cleaning
Getting rid of duplicates and null data, for cleaner data used for fine tuning

In [16]:
raw_data = raw_data.drop_duplicates()
raw_data.shape

(17877, 2)

In [21]:
raw_data = raw_data.dropna()
raw_data.shape

(17874, 2)

There are three classes, and 17874 rows of youtube comments data (with duplicates and nulls dropped), which should be good enough for fine-tuning use.

## Save into Directory

In [24]:
raw_data.to_csv("CleanYoutubeCommentsDataSet.csv")